src/Sentry/ISentryUserFactory.cs

```diff
@@ -0,0 +1,6 @@
+namespace Sentry;
+
+internal interface ISentryUserFactory
```

Comment:
@jamescrosswell I'm hitting the IUserFactory warning in our builds, and I went to do the cuover to silence them, but this interface is marked internal so I can't implement it in my projects.  Was this unintentional?

Reply:
Hey @jeremy-allocate , see https://github.com/getsentry/sentry-dotnet/pull/2618#issuecomment-1757367606 above. 

If it's an emergency and you just need to get a build out, you can disable the warning with a pragma, as we've done here:

https://github.com/getsentry/sentry-dotnet/blob/bbe4fa9d72b9c08e53b5bc33d07e2d52b2409b4f/src/Sentry.AspNetCore/Extensions/DependencyInjection/ServiceCollectionExtensions.cs#L26-L28

Reply:
All good, thanks @jamescrosswell.  We'll live with the warnings until #2714 is released.